django-data/image/validation/tasks.py

```diff
@@ -426,10 +491,52 @@ def submission_fail(self, submission_obj, message, status=NEED_REVISION):
         submission_obj.status = status
         submission_obj.message = ("Validation got errors: %s" % (message))
         submission_obj.save()
-
-        # send message with channel
         self.send_message(status, submission_obj)
 
+    def create_validation_summary(self, submission_obj,
+                                  submission_statuses_animals,
+                                  submission_statuses_samples):
+        """
+        This function will create ValidationSummary object that will be used
+        on validation_summary view
+        Args:
+            submission_obj: submission ref which has gone through validation
```

Comment:
You could also declare the type in sphinx, for instance:
```submission_obj (image_app.models.Submission): submission ref which has ...``` in `Args` while 
```:py:class:`Submission <image_app.models.Submission>``` in helper text

Reply:
Done